modules/ROOT/pages/_partials/endpoint-http-send.adoc

```diff
@@ -0,0 +1,70 @@
+== HTTP Send Settings
+
+These settings are a subset of those in xref:3.8@mule-runtime::http-request-connector.adoc[HTTP Request Connector].
```

Comment:
Why point to 3.8 doc?

Reply:
Ref to an earlier comment about HTTP settings. This is a placeholder version and the new version is not ready yet.